src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,67 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Movie birdman = new Movie();
+        birdman.setTitle("Birdman");
+        birdman.setDescription("Oscar 2015 - Best Picture");
+        Movie threeBillboards = new Movie();
+        threeBillboards.setTitle("Three Billboards Outside Ebbing, Missouri");
+        threeBillboards.setDescription("Oscar 2018 - Best Actress in a Leading Role");
```

Comment:
why are you using an empty constructor instead of a constructor with the given parameters?